dev/scripts/buildArktypeIO.ts

```diff
@@ -3,8 +3,6 @@ import { rmSync, writeFileSync } from "node:fs"
 import { join } from "node:path"
 import * as process from "node:process"
 
-// workaround for incompatible CJS/ESM imports (we have to use CJS in this dir because of docusaurus)
-//@blockFrom:dev/runtime/shell.ts:shell
 export type ShellOptions = Parameters<typeof execSync>[1] & {
```

Comment:
This should be deleted too and imported from runtime/shell.ts or whatever.

The only reason we were doing this copy block stuff was the cjs/mjs incompatibility, but like this it's just duplicated.